CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # CHANGELOG
 
+## [v2.29.3 _(Nov 21, 2022)_](https://github.com/omise/omise-magento/releases/tag/v2.29.3)
```

Comment:
Let change the version number to `2.30.0` rather than increasing the PATCH version because we have added new refund feature in this release.